f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -149,7 +149,12 @@
         'esd_auto_refresh',
         default=True,
         help='Enable ESD file periodic refresh'
-    )
+    ),
+    cfg.StrOpt(
+        'f5_extended_profile_name',
```

Comment:
The option name is a little long, right? Is it ok to use 'f5_extended_profile' as the option name?

Reply:
done